### PR TITLE
Create README

### PR DESCRIPTION
**Pull request by karenhlai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Frontend: React, Redux
+
+UI/UX
+The goal is to maintain a minimalistic design that will allow users to create many event-groups, while sustaining an organized dashboard. 
+- The app will have a splash page with animations of pictures of past Coachella events, as well as buttons to allow users to Signup or Login through a modal
+- Upon Signing up or Logging in, the user's dashbaord will render with event-groups they are participated in
+The event-groups will render in a chronological order starting with the next upcoming event. Only one event-group will appear on the dashboard at a time, while the adjacent listed event-groups will fade in upon scrolling in their direction, and fade out when no longer shown. 
+  Additional features: 
+  - scrollbar-less scrolling
+  - Each page has enough information
```
Comment (dowinterfor6) [inline README.md]: Probably delete this line

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - Borderless
+  - 100 uh/mm pages
```
Comment (dowinterfor6) [inline README.md]: 100 vh/vw

**Review by dowinterfor6 — CHANGES_REQUESTED** [review]
Change additional features, get rid of "Has enough information" and change "100uh/mm" to "100vw/vh"

**Review by dowinterfor6 — CHANGES_REQUESTED** [review]
Credit me for doing user auth :(

**Review by dowinterfor6 — APPROVED** [review]

**Comment by karenhlai** [comment]
lol oops changed it sry u deserve it, will update as we go as well ~

**Pull request merged**